Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Historical nicknames

## Developer changelog:
- Add old CK2 nicknames.
- Readjust them, placing a few in lifestyle events.
- Created two events for the title "the Old Murkeye" for the murlocs and for the title "the Old" for the humans.


# Where find the changes
- Warcraft-Guardians-of-Azeroth-2\common\nicknames _in wc_nicknames.txt_ **for the list of mod nicknames.**
- Warcraft-Guardians-of-Azeroth-2\events\lifestyles\warfare_lifestyle _in martial_strategy_event.txt_ **for the Mighty and the Ice-Hearted**
- Warcraft-Guardians-of-Azeroth-2\events\nickname_events _in nicknames_events.txt_ **for the Seeker**
- Warcraft-Guardians-of-Azeroth-2\events\nickname_events _in wc_nicknames_events.txt_ **for the new events for the Old Murkeye and the Old**
- Update localization here: Warcraft-Guardians-of-Azeroth-2\localization\english\event_localization\nickname (localization made by TheDrPibb)
# How to test:
- Open the command console
- Add nickname manually or launch an event. (Be careful, for many people you have to have the necessary conditions to see the new stuff. For example to see the option for "the Mighty" you need at least 15 martial, the strong trait and the ambitious trait.)
